### PR TITLE
Support python version 3.5

### PR DESCRIPTION
**Pull request by ondrejtomas**

### CODE REVIEW

File: ci/unit_tests/dialog_json2xml/main_test.py [hunk 1]
```diff
@@ -44,13 +46,16 @@ def test_mainValidActions(self):
...
         self.t_noException([[inputJsonPath, '-d', outputXmlDirPath]])
 
-        expectedXml = ""
-        outputXml = ""
-
         with open(expectedXmlPath, 'r') as expectedXmlFile:
-            expectedXml = expectedXmlFile.read()
+            expectedXml = etree.XML(expectedXmlFile.read(), etree.XMLParser(remove_blank_text=True))
+            for parent in expectedXml.xpath('//*[./*]'): # Search for parent elements
+                print(parent)
```
Comment (ota-sandr) [inline ci/unit_tests/dialog_json2xml/main_test.py]: Remove this print to avoid std-out spoiling

File: ci/unit_tests/dialog_json2xml/main_test.py [hunk 2]
```diff
@@ -44,13 +46,16 @@ def test_mainValidActions(self):
...
+                parent[:] = sorted(parent,key=lambda x: x.tag)
```
Comment (ota-sandr) [inline ci/unit_tests/dialog_json2xml/main_test.py]: Missing space after comma

File: ci/unit_tests/dialog_xml2json/main_test.py [hunk 3]
```diff
@@ -54,5 +54,5 @@ def test_mainValidActions(self):
         with open(outputJsonPath, 'r') as outputJsonFile:
             outputJson = outputJsonFile.read()
 
-        assert expectedJson == outputJson
+        assert json.loads(expectedJson) == json.loads(outputJson)
```
Comment (ota-sandr) [inline ci/unit_tests/dialog_xml2json/main_test.py]: You can use `json.load` directly on file-like object.

File: ci/unit_tests/dialog_json2xml/main_test.py (hunk 1 above)
Comment (ondrejtomas) [reply ci/unit_tests/dialog_json2xml/main_test.py]: ok, thanks :)

File: ci/unit_tests/dialog_json2xml/main_test.py (hunk 2 above)
Comment (ondrejtomas) [reply ci/unit_tests/dialog_json2xml/main_test.py]: ok, thanks :)

File: ci/unit_tests/dialog_xml2json/main_test.py (hunk 3 above)
Comment (ondrejtomas) [reply ci/unit_tests/dialog_xml2json/main_test.py]: ok, thanks :)

**Comment by Tereza-Pytelova** [comment]
This was splitted to #328 and #329

**Pull request closed without merging**